### PR TITLE
Refactoring - Remove data from src + remove numba dependency

### PR DESCRIPTION
**Pull request by artzet-s**

### CODE REVIEW

**Comment by artzet-s** [comment]
close #12
close #2

Waiting for @pradal review

**Comment by pradal** [comment]
OK. I will do that ... tomorrow.

**Comment by artzet-s** [comment]
@pradal Don't forget !

File: .readthedocs.yml [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-  file: doc/environment_list.txt
+  file: doc/environment.yml
 
 build:
   image: latest
 
 python:
-  setup_py_install: True
\ No newline at end of file
+  setup_py_install: false
```
Comment (pradal) [inline .readthedocs.yml]: Add an empty line at the eof

File: .travis.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@ install:
   - cd travis-ci
   - source install.sh
   - conda config --prepend channels conda-forge
+  - conda install -y conda-verify
```
Comment (pradal) [inline .travis.yml]: why do you use conda-verify?

File: README.rst [hunk 3]
```diff
@@ -75,21 +64,68 @@ Please, cite the software, if you are using for your research.
...
+
+From source
+-----------
+
+.. code::
+
+    # Install dependency with conda
+    conda create -n phm -c conda-forge python=3
+    conda activate phm
+    conda install -c conda-forge -c openalea cython numpy numba scipy scikit-image scikit-learn networkx=2.3 opencv matplotlib vtk pytest
```
Comment (pradal) [inline README.rst]: Why networkx 2.3? We can add an issue if we want to use the latest version on conda-forge

File: doc/environment.yml [hunk 4]
```diff
@@ -5,17 +5,4 @@ channels:
...
-      - scikit-image
-      - scipy
-      - numba
-      - numpy
-      - cython
-      - matplotlib
-      - sphinx
-      - sphinx_rtd_theme
\ No newline at end of file
+  - openalea.phenomenal
```
Comment (pradal) [inline doc/environment.yml]: add newline

File: src/openalea/phenomenal/data/data.py [hunk 5]
```diff
@@ -38,99 +37,93 @@ def _path_images(plant_number=1, dtype="bin"):
...
-            plant_number, dtype))
+    data_directory = os.path.join(name_dir, '{}/'.format(dtype))
 
     d = collections.defaultdict(dict)
     for id_camera in ["side", "top"]:
-        filenames = glob.glob(os.path.join(data_directory, id_camera, '*.png'))
+        filenames = glob.glob(os.path.join(data_directory, id_camera, '*'))
         for filename in filenames:
-            angle = int(os.path.basename(filename).split('.png')[0])
+            angle = int(os.path.basename(filename)[:-4])
```
Comment (pradal) [inline src/openalea/phenomenal/data/data.py]: You can get the extension (.jpeg is 5 char)

File: README.rst (hunk 3 above)
Comment (artzet-s) [reply README.rst]: ok

Comment (artzet-s) [reply README.rst]: #21

File: .readthedocs.yml (hunk 1 above)
Comment (artzet-s) [reply .readthedocs.yml]: done with last commit

File: src/openalea/phenomenal/data/data.py (hunk 5 above)
Comment (artzet-s) [reply src/openalea/phenomenal/data/data.py]: done with last commit

File: doc/environment.yml (hunk 4 above)
Comment (artzet-s) [reply doc/environment.yml]: done with last commit

**Comment by artzet-s** [comment]
close #8

**Pull request merged**